precompiles/referenda/src/lib.rs

```diff
@@ -275,7 +271,28 @@ where
 
 		<RuntimeHelper<Runtime>>::try_dispatch(handle, Some(origin).into(), call)?;
 
-		Ok(())
+		Ok(referendum_index)
+	}
+
+	/// Propose a referendum on a privileged action.
+	///
+	/// Parameters:
+	/// * track_id: The trackId for the origin from which the proposal is to be dispatched.
+	/// * proposal: The proposed runtime call.
+	/// * block_number: Block number at which proposal is dispatched.
+	#[precompile::public("submitAt(uint16,bytes,uint32)")]
+	fn submit_at(
```

Comment:
In the Solidity file `submitAt` and `submitAfter` returns nothing, while in Rust they return `u32`.